frontend/src/domain/WeclomeScreen/WelcomeScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { Modal, Form } from "./Form";
+import { Welcome } from "./Welcome/Welcome";
+import { Props, mapTypeToFormProps } from "./utils";
+import { theme } from "../../constants";
+
+const Purple = styled.div`
+  padding-left: 0.5rem;
+  color: ${theme.colors.mediumOrchid};
+`;
+
+const mapTypeToTitle = (type: Props["type"]) => {
```

Comment:
Or maybe `utils.js` file would be better place for the function? 🤔

Reply:
moved